examples/jsm/nodes/accessors/BitangentNode.js

```diff
@@ -5,9 +5,9 @@ import { tangentGeometry, tangentLocal, tangentView, tangentWorld, transformedTa
 
 const getBitangent = ( crossNormalTangent ) => crossNormalTangent.mul( tangentGeometry.w ).xyz;
 
-export const bitangentGeometry = varying( getBitangent( normalGeometry.cross( tangentGeometry ) ) ).normalize();
-export const bitangentLocal = varying( getBitangent( normalLocal.cross( tangentLocal ) ) ).normalize();
-export const bitangentView = varying( getBitangent( normalView.cross( tangentView ) ) ).normalize();
-export const bitangentWorld = varying( getBitangent( normalWorld.cross( tangentWorld ) ) ).normalize();
-export const transformedBitangentView = getBitangent( transformedNormalView.cross( transformedTangentView ) ).normalize();
-export const transformedBitangentWorld = transformedBitangentView.transformDirection( cameraViewMatrix ).normalize();
+export const bitangentGeometry = /*#__PURE__*/ varying( getBitangent( normalGeometry.cross( tangentGeometry ) ), 'v_bitangentGeometry' ).normalize().toVar( 'bitangentGeometry' );
```

Comment:
What's the difference between `varying()` and `toVar()`?

Reply:
`toVar( name = null )` will create a variable, the first parameter is used to add a name to it, otherwise the node system will name it automatically, it can be useful in debugging or access using `wgslFn`.

`varying( node, name = null )` this node is very interesting, let's suppose you want to optimize some calculation in the `vertex stage` but are using it in a slot like `material.colorNode`, with `varying` this is possible. For example:

```js
// multplication will be executed in vertex stage
const inVertexStage = varying( modelNormalMatrix.mul( normalLocal ) );

// normalize will be executed in fragment stage
// because .colorNode is a fragment stage slot as default
material.colorNode = inVertexStage.normalize();
```

The first parameter of `varying( modelNormalMatrix.mul( normalLocal ) )`  will be executed in `vertex stage`, and the return from `varying()` will be a `varying` as we are used in WGSL/GLSL, this can optimize extra calculations in the `fragment stage`. The second parameter allows you to add a custom name to `varying`.

If `varying()` is added only to `.positionNode`, it will only return a simple variable and varying will not be created.

Reply:
Sounds good! Thanks for explaining 🙏

I've started a [wiki page for TSL](https://github.com/mrdoob/three.js/wiki/TSL-Spec).
Any chance you can spend some time completing it?